python_testing: Add Checks for Command IDs

soares-sergio opened the pull request:
This PR adds checks and unit tests for the command ID ranges defined in Table 79 of the spec.
These checks will be used on the various Python tests, for example, `TC_DeviceBasicComposition.py`.

soares-sergio committed:
python_testing: Add Checks for Command IDs

This PR adds checks and unit tests for the command ID ranges defined in Table 79 of the spec.
These checks will be used on the various Python tests, for example,
TC_DeviceBasicComposition.py.

restyled-commits committed:
Restyled by isort

tcarmelveilleux commented on src/python_testing/global_attribute_ids.py:
Perhaps this should now be called `global_ids.py` since it's not just attribute IDs

soares-sergio committed:
python_testing: Update command id error msg

This PR uses the newly added command id check.
It updates the error msg to indicate if it's a "Test Vendor" id.

soares-sergio committed:
Merge branch 'master' into add_command_id_checks

soares-sergio committed:
Merge branch 'master' into add_command_id_checks

soares-sergio committed:
Merge remote-tracking branch 'upstream/master' into add_command_id_checks

soares-sergio committed:
fix merge conflicts

soares-sergio commented:
Moved the changes from here to another PR so they merge together I don't have to sheperd 2 separate ones: https://github.com/project-chip/connectedhomeip/pull/36018

restyled-commits committed:
Restyled by isort
